python/hsfs/training_dataset.py

```diff
@@ -619,7 +619,9 @@ def query(self):
         """Query to generate this training dataset from online feature store."""
         return self._training_dataset_engine.query(self, True, True, False)
 
-    def get_query(self, online: bool = True, with_label: bool = False, is_hive_query: bool = False):
+    def get_query(
+        self, online: bool = True, with_label: bool = False, is_hive_query: bool = False
```

Comment:
We don't actually want to surface this to the user, the user is not necessarily aware of what engine he is using since it's all transparent. E.g. it will use Spark if available otherwise not.

So you can get the engine type from the engine singleton like this:
```
from hsfs import engine
engine.get_type()
```